api/camunda/camunda-openapi.yaml

```diff
@@ -25,11 +25,34 @@ servers:
         default: http
         description: The schema of the Camunda 8 REST API server.
 
+tags:
+  - name: Authorization
+  - name: Clock
+  - name: Cluster
+  - name: Decision definition
+  - name: Decision instance
+  - name: Decision requirements
+  - name: Document
+  - name: Element instance
+  - name: Flow node instance
+  - name: Incident
+  - name: Job
+  - name: License
+  - name: Message
+  - name: Process definition
+  - name: Process instance
+  - name: Resource
+  - name: Signal
+  - name: User
+  - name: User task
+  - name: Variable
+
 paths:
   /topology:
     get:
       tags:
         - Cluster
+      operationId: getTopology
```

Comment:
Surprisingly, adding operationIds to the endpoints causes them to generate in a different file path. This isn't a problem, just not something I was expecting. 

I will make a note for myself in the htaccess as part of this PR, pointing back here, because at time of the next release I will need to create a bunch of redirect rules for everything that moved.

Reply:
Oh yeah, I should have given you a heads-up, I missed that. The long-term benefit with this is: the file path and URL is now based on the operation id instead of the path summary (which can easily change). Thus, URLs will be more stable from now on.